DNN Platform/Library/Entities/Portals/PortalSettings.cs

```diff
@@ -590,16 +589,7 @@ public bool IsThisPortalLocked
             get { return PortalController.GetPortalSettingAsBoolean("IsLocked", PortalId, false); }
         }
 
-        public TimeZoneInfo TimeZone
-        {
-            get { return _timeZone; }
-            set
-            {
-                _timeZone = value;
-                PortalController.UpdatePortalSetting(PortalId, "TimeZone", value.Id, true);
-            }
-        }
-
+        public TimeZoneInfo TimeZone { get; set; }

```

Comment:
Should we continue defaulting this to `TimeZoneInfo.Local`?
```suggestion
        public TimeZoneInfo TimeZone { get; set; } = TuneZoneInfo.Local;
```

Reply:
Watch out, you have TuneZone instead of TimeZone in your suggested change here...